client/public/index.html

```diff
@@ -2,6 +2,23 @@
     <head>
         <meta charset="UTF-8">
         <meta name="viewport" content="width=device-width, initial-scale=1.0">
+        <title>Scaffolding for full stack React - Polaris 2018 | Opensource</title>
+        <meta name = "description" content="Welcome to Polaris 2018 - Best scaffolding for React using full stack JavaScript technologies | Polaris 2018 | Opensource | Scaffolding | Full Stack Javascript | React | TypeScript | Apollo | GraphQL | Jest | Express | React Router" />
+
+        <!--  Essential META Tags -->
+        <meta property="og:title" content="Polaris 2018 - Opensource Scaffolding for full stack React">
+        <meta property="og:description" content="Best scaffolding for React using full stack JavaScript technologies">
+        <meta property="og:image" content="https://polaris2018.com/thumbnail.jpg">
```

Comment:
We can use this image here: https://github.com/tanaypratap/polaris-2018/blob/master/client/public/img/icon-72.png